public/pages/utils/anomalyResultUtils.ts

```diff
@@ -124,32 +124,42 @@ export const prepareDataForChart = (
   if (anomalies.length > MAX_DATA_POINTS) {
     anomalies = sampleMaxAnomalyGrade(anomalies);
   }
-
-  anomalies.push({
-    startTime: dateRange.startDate,
-    endTime: dateRange.startDate,
-    plotTime: interval
-      ? dateRange.startDate - MIN_IN_MILLI_SECS * interval
-      : dateRange.startDate,
-    confidence: null,
-    anomalyGrade: null,
-  });
-  anomalies.unshift({
-    startTime: dateRange.endDate,
-    endTime: dateRange.endDate,
-    plotTime: interval
-      ? dateRange.endDate + MIN_IN_MILLI_SECS * interval
-      : dateRange.endDate,
-    confidence: null,
-    anomalyGrade: null,
-  });
-  if (getFloorPlotTime) {
+  if (getFloorPlotTime) { // we need to get floor plot time for bar chart
```

Comment:
question: In which cases `getFloorPlotTime ` is undefined?

Reply:
As discussed offline, there exists case where `getFloorPlotTime ` is not needed, aka LineSeries. I am good with that.